core/src/main/scala/cats/collections/Heap.scala

```diff
@@ -180,9 +180,10 @@ sealed abstract class Heap[A] {
    * prefer unorderedFoldMap if you can express your operation as a commutative monoid
    * since it is O(N) vs O(N log N) for this method
    */
-  def foldLeft[B](init: B)(fn: (B, A) => B)(implicit order: Order[A]): B = {
+  // TODO ceedubs should this be (B, A) or (B, AA)?
```

Comment:
Since Function2 is contravariant AA is the more general type, since you could also accept A for that case.

Reply:
Oops I meant to grep for TODO before I pushed this. I'll clean this up.